packages/react-examples/src/experiments/UnifiedPeoplePicker/DoubleUnifiedPeoplePicker.Example.tsx

```diff
@@ -179,14 +161,54 @@ const UnifiedPeoplePickerExample = (): JSX.Element => {
     setPeopleSelectedItems(updatedItems);
   };
 
-  const _onInputChange = (filterText: string): void => {
+  const SEPARATOR_REGEX = new RegExp(';|,|\\n|\u000A', 'g');
+  const splitPasteInputIntoRecipientStrings = (inputText: string) =>
+    inputText
+      .split(SEPARATOR_REGEX)
+      .map((token: string) => token.trim())
+      .filter((token: string) => !!token);
+
+  const _onInputChange = (filterText: string, composing?: boolean, resultItemsList?: IPersonaProps[]): void => {
     const allPeople = people;
+
+    const lastCharIndex = filterText.length - 1;
+    const lastChar = filterText[lastCharIndex];
     const suggestions = allPeople.filter((item: IPersonaProps) => _startsWith(item.text || '', filterText));
     const suggestionList = suggestions.map(item => {
       return { item: item, isSelected: false, key: item.key } as IFloatingSuggestionItem<IPersonaProps>;
     });
-    // We want to show top 5 results
-    setPeopleSuggestions(suggestionList.splice(0, 5));
+
+    const updatedItems: IPersonaProps[] = [];
+    const currentItems: IPersonaProps[] = [...peopleSelectedItems];
+
+    for (let i = 0; i < currentItems.length; i++) {
```

Comment:
I don't think you need this, you can have the declaration be this right?
const updatedItems: IPersonaProps[] = [...peopleSelectedItems];